javascript/chronometer.js

```diff
@@ -4,33 +4,38 @@ class Chronometer {
     this.intervalId = 0;
 
   }
-  startClick(callback) {
-    setInterval(()=>{
-      this.currentTime++
-    },1000)
+  startClick() {
+    this.intervalId = setInterval(function(){ 
```

Comment:
Interval function needs to be an arrow function in order for the this keyword to target to the correct element (the classChronometer)